src/nn/ops/split.ts

```diff
@@ -38,6 +39,27 @@ export class Split extends Operation {
 
   computeImpl(inputTensors: Map<MLOperand, tf.Tensor>): tf.Tensor[] {
     const input: tf.Tensor = inputTensors.get(this.input_);
-    return tf.split(input, this.splits_, this.axis_);
+    const outputs = tf.split(input, this.splits_, this.axis_);
+    if (this.needCheckOutputShape_) {
+      const axis = this.axis_ >= 0 ? this.axis_ : this.axis_ + input.rank;
+      let sliceSizes = [];
+      if (typeof this.splits_ === 'number') {
+        sliceSizes =
+            new Array(this.splits_).fill(input.shape[axis] / this.splits_);
+      } else {
+        sliceSizes = this.splits_.slice();
+      }
+      const outputsShape = [];
+      for (const size of sliceSizes) {
+        const outputShape = input.shape.slice();
```

Comment:
Move this line out of the loop.

Reply:
@mingmingtasd The `outputShape` would be updated in `for` loop for each sliceSie, so it doesn't need to move out, thanks.

Reply:
> 

Need outputShape = input.shape.slice() be updated?